src/plots/gl3d/layout/axis_defaults.js

```diff
@@ -56,7 +56,15 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, options) {
             options.fullLayout);
 
         coerce('gridcolor', colorMix(containerOut.color, options.bgColor, gridLightness).toRgbString());
-        coerce('title', axName[0]);  // shouldn't this be on-par with 2D?
+        coerce('title.text', axName[0]);  // shouldn't this be on-par with 2D?
+
+        // TODO 882 coercing old 'titlefont' was missing. Why? Activating the below would break a lot image tests...
```

Comment:
No need to worry here. `title.font` (and so did `titlefont`) is coerced in `handleAxisDefaults` a couple lines above this line.

In fact, `title.text` (and so did `title`) is also coerced in `handleAxisDefaults`. This line here coerces it again with the "correct" gl3d default value.

Reply:
Okay, thanks. Removed it.